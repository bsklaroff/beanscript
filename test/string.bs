str = "asdf"

print(len(str))
print(str[0])
print(str[1])
print(str[2])
print(str[3])

str = "a'\"\"'\\b"
print(len(str))
print(str[0])
print(str[1])
print(str[2])
print(str[3])
print(str[4])
print(str[5])
print(str[6])
print(str[7])

str = 'q1'
print(len(str))
print(str[0])
print(str[1])

str = '0"\\\'\'"1'
print(len(str))
print(str[0])
print(str[1])
print(str[2])
print(str[3])
print(str[4])
print(str[5])
print(str[6])
print(str[7])
