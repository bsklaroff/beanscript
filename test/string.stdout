4
a
s
d
f
8
a
'
"
"
'
\
\
b
2
q
1
8
0
"
\
\
'
'
"
1
